domain/snyk/issues.go

```diff
@@ -75,10 +75,19 @@ type Issue struct {
 	GlobalIdentity string
 }
 
+func (i *Issue) GetFrom() []string {
+	ossIssueData, ok := i.AdditionalData.(OssIssueData)
+	if !ok {
+		return nil
+	}
+	return ossIssueData.From
+}
+
 var _ delta.Identifiable = (*Issue)(nil)
 var _ delta.Fingerprintable = (*Issue)(nil)
 var _ delta.Locatable = (*Issue)(nil)
 var _ delta.Pathable = (*Issue)(nil)
+var _ delta.Fromable = (*Issue)(nil)
```

Comment:
I'd like something more specific, that does not imply knowledge of the json output and its field names - something like "introducedBy" maybe?